xep-0065.xml

```diff
@@ -776,7 +786,7 @@ DATA = (payload)
     <p>The "jid" attribute specifies the JID of the StreamHost. This attribute MUST be present, and MUST be a valid JID for communication over XMPP.</p>
   </section2>
   <section2 topic='&lt;activate/&gt; Element' anchor='desc-activate'>
-    <p>The &lt;activate/&gt; element is sent from the Requester to the Proxy in order to formally start the bytestream. This element is always empty and has no defined attributes.</p>
+    <p>The &lt;activate/&gt; element is sent from the Requester to the Proxy in order to formally start the bytestream. This element has no defined attributes and its XML text contains always the JID of the target.</p>
```

Comment:
That's the term from the XML specification and we use it everywhere. If it's confusing, let's complain to the W3C rather than make up our own terminology. :-)